src/dune_engine/dune_project.ml

```diff
@@ -717,6 +728,9 @@ let parse ~dir ~lang ~file ~dir_status =
         and+ wrapped_executables =
           field_o_b "wrapped_executables"
             ~check:(Dune_lang.Syntax.since Stanza.syntax (1, 11))
+        and+ map_workspace_root =
+          field_o_b "map_workspace_root"
+            ~check:(Dune_lang.Syntax.since Stanza.syntax (3, 6))
```

Comment:
Should be `(3, 7)`

Reply:
@rgrinberg, I have pushed to correct this, and also in the documentation.